### PR TITLE
Interpolate access levels by key in vpc sc module

### PR DESCRIPTION
**Pull request by ludoo**

### CODE REVIEW

**Comment by ludoo** [comment]
WIP

File: modules/vpc-sc/README.md [hunk 1]
```diff
@@ -122,7 +130,7 @@ module "test" {
     r1 = {
       spec = null
       status = {
-        access_levels       = [module.test.access_level_names["a1"]]
+        access_levels       = [module.test.access_level_names["a1"], "a2"]
```
Comment (sruffilli) [inline modules/vpc-sc/README.md:133]: access_levels = [module.test.access_level_names["a1"], module.test.access_level_names["a2"]] ?

Comment (sruffilli) [reply modules/vpc-sc/README.md:133]: aaaah right!

Comment (ludoo) [reply modules/vpc-sc/README.md:133]: lol, the point of this PR is exactly to do what is shown above...

**Pull request merged**